iaf-aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/services/util/DateUtil.java

```diff
@@ -0,0 +1,65 @@
+package nl.nn.adapterframework.extensions.aspose.services.util;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+/**
+ * @author Gerard van der Hoorn(d937275)
+ * 
+ */
+public final class DateUtil {
+
```

Comment:
Most of these functions already exist in the framework. It would be nice if they wouldn't be duplicated